### ISSUE TITLE
MaterialRatingBar is not showing on ListView Item placed in DrawerLayout NavigationView.

### ISSUE THREAD

**Issue by Maqsood007** [DreaminginCodeZH/MaterialRatingBar#15]
I have placed the xml Of MaterialDesignRatingBar in a expandible lisview item and it is not showing at all. its working on my other listviews with in Activities and fragments.

 <me.zhanghai.android.materialratingbar.MaterialRatingBar
                android:id="@+id/ratingBar"
                style="@style/Widget.MaterialRatingBar.RatingBar"
                android:layout_width="@dimen/_150sdp"
                android:layout_height="@dimen/_25sdp"
                android:layout_centerInParent="true"
                android:rating="5"
                android:layout_marginLeft="@dimen/_5sdp"
                android:layout_marginStart="@dimen/_5sdp" />  

I don't want to go for another custom library for rating bar because i have used it in my project.

**Comment by zhanghai** [comment]
If you replace it with framework `RatingBar`, does it work? If not, it is a usage issue of `RatingBar` instead of a library issue.

**Comment by Maqsood007** [comment]
Yes it works and rating stars are showing up with framework RatingBar.

**Comment by zhanghai** [comment]
Well it's likely a library issue then. Can you provide a [minimal working example](https://stackoverflow.com/help/mcve) project?

**Comment by Maqsood007** [comment]
Any progress on this issue ?

**Comment by zhanghai** [comment]
Could you pleade provide a [MWE](https://stackoverflow.com/help/mcve) for this?

**Comment by BigCowHere** [comment]
Yes ,the same as RecycleView item

**Comment by dcmouser** [comment]
I'm having a similar problem, MaterialRatingBar is not showing anything when in listview.

**Comment by Hynn01** [comment]
I have the same problem on using it on RecycleView...

**Comment by ss-southernsun-982** [comment]
I have the same problem when trying to put the MaretialRatingBar on Recyclerview, when i replaced it with the default RatingBar, it works perfectly
